modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,25 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+When you request access, the client application is registered and creates an
```

Comment:
To implement an API, the client application must first request access. When the request is approved by the API, a contract is created between the client application and the API, and the client application is registered.

Reply:
Updated to use this text. Thank you!